include/ode_solvers/oracle_functors_rcpp.hpp

```diff
@@ -0,0 +1,120 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+// Copyright (c) 2020-2020 Marios Papachristou
+
+// Contributed and/or modified by Marios Papachristou, as part of Google Summer of Code 2020 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef ORACLE_FUNCTORS_RCPP_HPP
+#define ORACLE_FUNCTORS_RCPP_HPP
+
+enum ode_solvers {
+  no_solver,
```

Comment:
what happens in the `no_solver` case ? `rk` could have been `range-kutta`

Reply:
The no_solver case takes care of the case that we do uniform sampling. 
the elements of this enum are used for the sample_points() function of the R API
and the ode_solve function. We want from parsing the user arguments supplied
to sample_points to do a switch regarding which solver to use.

Reply:
Also changed `rk` to `runge-kutta`